src/utils/certify.js

```diff
@@ -9,14 +9,21 @@ var certifiedTokensList = {
         'TWtSxBEx7rmsQ34MyWzwBCYYwRJh4K9xsL9zPkMK8': 'DLL'
     },
 }
-
-import { NETWORK_BYTE } from '@/network'
+import Blockchain from '@/js-v-sdk/src/blockchain'
+import { NETWORK_BYTE, NODE_IP } from '@/network'
 var certifiedTokens;
+var certifiedTokensInfo = {};
+var chain = new Blockchain(NODE_IP, NETWORK_BYTE);
 if (String.fromCharCode(NETWORK_BYTE) === 'T') {
     certifiedTokens = certifiedTokensList['Testnet']
 } else {
     certifiedTokens = certifiedTokensList['Mainnet']
 }
+for (let token in certifiedTokens) {
+    chain.getTokenInfo(token).then(response => {
```

Comment:
Only Split-Type Tokens need call `chain.getTokenInfo` to get latest unity. The others just load unity from config.